### PR TITLE
Add Match Documentation to User Guide

### LINKED ISSUE #92: Add Match Documentation to User Guide

### PR DESCRIPTION
**Pull request by adidoesnt**

### CODE REVIEW

**Comment by VisnuRavi** [comment]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -83,20 +83,35 @@ Examples:
   Returns all articles of clothing that are white in colour and have either sizes 36 or 23.
   <br><br>
   
-**7. Matching...: `match`**
-<br><br>
+**8. Matching multiple garments to create an outfit: `match`**<br>
```
Comment (Jonathan-Cao) [inline docs/UserGuide.md:86]: Detailed documentation

**Review by VisnuRavi — COMMENTED** [review]
Nice Job!

**Review by Jonathan-Cao — APPROVED** [review]
LGTM

**Review by VisnuRavi — APPROVED** [review]
LGTM

**Pull request merged**